templates/guides/vault-scaling.md.tmpl

```diff
@@ -0,0 +1,19 @@
+---
+subcategory: ""
+page_title: "Resize or scale a Vault cluster - HCP Provider"
+description: |-
+    Change a current HCP Vault cluster in terms of tiers (Dev, Starter, Standard) or sizes (S, M, L).
+---
+
+# Resize or scale a cluster
```

Comment:
Currently the tier consists of `tier_size` and it's not really possible to change one without the other. I wonder if we should simplify the language here to just `scale`. Right now, seeing `change size OR tier` makes me think I should be able to change them separately.
```suggestion
# Scale a cluster
```